solr_configs/schema.xml

```diff
@@ -30,10 +30,6 @@
     <fieldType name="float" class="solr.TrieFloatField" precisionStep="0" omitNorms="true" sortMissingLast="true" positionIncrementGap="0"/>
```

Comment:
Just a note: several of these fields are deprecated in Solr 7

http://lucene.apache.org/solr/7_1_0/solr-core/deprecated-list.html